app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -16,14 +16,31 @@
 
   intervention_helper = GovukPublishingComponents::Presenters::InterventionHelper.new(local_assigns)
   dismiss_href = intervention_helper.dismiss_link
+
+  suggestion_tag_options = {
+    class: "govuk-link gem-c-intervention__suggestion-link",
+    href: suggestion_link_url,
+    data: suggestion_data_attributes,
+  }
+
+  new_tab ||= false
+  if new_tab && (suggestion_link_text && suggestion_link_text)
```

Comment:
I'd suggest moving most of this code into the intervention component helper.

Reply:
Done